stories/widgets.stories.js

```diff
@@ -41,37 +42,35 @@ const { HALF, QUARTER, FULL } = WIDGET_WIDTHS;
 
 function BoxesWidgets( { children } ) {
 	return (
-		<div className="mdc-layout-grid googlesitekit-widget-area googlesitekit-widget-area--boxes">
+		<Grid className="googlesitekit-widget-area googlesitekit-widget-area--boxes">
 			<div className="googlesitekit-widget-area-widgets">
-				<div className="mdc-layout-grid__inner">{ children }</div>
+				<Row>{ children }</Row>
 			</div>
-		</div>
+		</Grid>
 	);
 }
 
 function CompositeWidgets( { children } ) {
 	return (
-		<div className="mdc-layout-grid googlesitekit-widget-area googlesitekit-widget-area--composite">
+		<Grid className="googlesitekit-widget-area googlesitekit-widget-area--composite">
 			<div className="googlesitekit-widget-area-widgets">
```

Comment:
I think we can combine this div with the following `Row` element to be `<Row className="googlesitekit-widget-area-widgets">`. Can you try this? If there are no visual changes, then let's update it to remove this div between `Grid` and `Row` elements.

Reply:
Good shout, this works and I've removed the `div` (and repeated for the `BoxesWidgets` function above).